src/Bundle/CoverallsBundle/Collector/CiEnvVarsCollector.php

```diff
@@ -110,6 +111,34 @@ protected function fillTravisCi()
         return $this;
     }
 
+    /**
+     * Fill Github Actions environment variables.
+     *
+     * @return $this
+     */
+    protected function fillGithubActions()
+    {
+        if (isset($this->env['GITHUB_ACTIONS'])) {
```

Comment:
I'd invert the condition and exit early.

Reply:
What do you mean say about inverted condition?

Reply:
`if (!isset($this->env['GITHUB_ACTIONS'])) { return $this; }`

Reply:
@simPod, I see on your changes [here](https://github.com/simPod/php-coveralls/commit/b9c9d4c8ffdf837c589aba5da3f83e660350bfea) and perhaps take your code

Reply:
Yes, you can merge the PR into your branch. In that way my commit propagates here and you can continue on top of it.

Reply:
I merge your PR, thanks for your job!